.travis.yml

```diff
@@ -1,14 +1,30 @@
 language: csharp
 
+sudo: required
+dist: trusty
+
+env:
+  - CLI_VERSION=1.0.0-preview1-002702
+
+addons:
+  apt:
+    packages:
+    - gettext
+    - libcurl4-openssl-dev
+    - libicu-dev
+    - libssl-dev
+    - libunwind8
+
```

Comment:
Well they may well be required by ASP.NET but not by just the core. I suspect all of this will change over time, to be honest.